package.json

```diff
@@ -32,6 +32,7 @@
     "mrmlnc.vscode-duplicate",
     "neptunedesign.vs-slug",
     "Prisma.prisma",
+    "styled-components.vscode-styled-components",
```

Comment:
Bump the version to '0.0.3' to automatically publish it on merge

Reply:
bumped